CharacterCreation/03_Feats.txt

```diff
@@ -190,6 +190,11 @@ lower chance to hit than the previous hit. (A 4th level character making their
 applies to a monk's Shadow Strike as well, increasing the DC by 20 per strike 
 instead of increasing the strike's miss chance.
 
+Last Stand:
+Prerequisite: Level 4
+Don't die alone. Always bring a friend. You may take an action to do one of 
+the following: Draw a one-handed weapon such as a pistol or SMG, Aim, or fire. 

```

Comment:
@bleehu I hope you won't lambaste me for saying this since we've had recent drama about writing styles (I have good intentions):
I think it would be more clear to the reader if the latter half of the last sentence gets rewritten from: 

>Draw a one-handed weapon such as a pistol or SMG, Aim, or fire. 

to

>Draw, Aim, or Fire a one-handed weapon such as a pistol or SMG.


Also, do you intend for the feat to allow you to take one action per turn or just one action per Down, **period**?
Also, you removed the part that explains what triggers this ability.

Reply:
"After each successful passing of the fort save for the "downed" condition," is what i wrote for the trigger, and going off of basic rules: " You roll one saving throw per turn 
that you are downed and these DSTs are not modified by any attribute" 

Meaning if you succeed your death saving throw and have this feat, you can do something small as defined in the feat instead of just losing your turn.

Reply:
change fort to dst, maybe add the only dst modifier to it? maybe add 10% making a roll of 40%?

Reply:
would have to update basic rules.

Reply:
@bleehu please respond to my questions, above.